cpp_api/api.cpp

```diff
@@ -0,0 +1,65 @@
+#include "api.h"
+
+#include <util/cmdline.h>
+#include <util/config.h>
+#include <util/message.h>
+#include <util/options.h>
+
+#include <goto-programs/goto_model.h>
+#include <goto-programs/initialize_goto_model.h>
+
+#include <ansi-c/ansi_c_language.h>
+#include <langapi/mode.h>
+
+#include <memory>
+#include <options.h>
+
+extern configt config;
+
+// Default initialise the api_dependencies object
+cbmc_api::api_dependenciest::api_dependenciest()
+  : // We don't print anything to the console as part of the API
+    message_handler(new null_message_handlert()),
```

Comment:
🚫 Unique pointers should be constructed via `util_make_unique` rather than directly using `new` like this.